meinberlin/assets/scss/_base.scss

```diff
@@ -127,8 +127,9 @@ dt {
     margin-top: $em-spacer;
 }
 
-button,
-[type="submit"] {
+// FIXME what was this for? breaks button styling?!
+// [type="submit"]
+button {
```

Comment:
this will now apply to all buttons (if there are others?) instead of just the ones which do a submit, do we want that?